internal/github/checkPendingCI.go

```diff
@@ -14,11 +14,11 @@ import (
 )
 
 // CheckForPendingCI returns a list of PR URLs that are ready for CI to run but haven't started yet.
-func CheckForPendingCI(githubClient *github.Client, internalTeam string, cfg config.CheckStalePending) ([]string, error) {
+func CheckForPendingCI(githubClient *github.Client, internalTeam string, repos *config.Config) ([]string, error) {
```

Comment:
I'd maybe consider renaming this to `cfg *config.Config` or something like that, because when you call on it in the function below, it's not super clear that there's more to this struct than just repo data. But since it actually contains data about users and all that, the name "repos" is a little misleading imo.

Reply:
fixed